hack/build-manifests.sh

```diff
@@ -32,8 +32,33 @@ for arg in $args; do
     final_out_dir=$(dirname ${MANIFESTS_OUT_DIR}/${arg})
     mkdir -p ${final_out_dir}
     manifest=$(basename -s .in ${arg})
+    outfile=${final_out_dir}/${manifest}
     sed -e "s#{{ docker_tag }}#${docker_tag}#g" \
         -e "s#{{ docker_prefix }}#${manifest_docker_prefix}#g" \
         -e "s#{{ namespace }}#${namespace}#g" \
-        ${KUBEVIRT_DIR}/manifests/$arg >${final_out_dir}/${manifest}
+        ${KUBEVIRT_DIR}/manifests/$arg >${outfile}
+
+    set +e
+    grep -q '^.*APPEND_AUTOGENERATED_VM_CRD$' $outfile
```

Comment:
Can we start using jinja again or go templates?

Reply:
possibly, i'll look into it.